README.md

```diff
@@ -70,6 +70,61 @@ the user needs to provide very minimal configuration for monitoring any type of
 
 Luminaire can also monitor a set of data points over windows of time instead of tracking individual data points. This approach is well-suited for streaming use cases where sustained fluctuations are of greater concern than individual fluctuations. See [anomaly detection for streaming data](https://zillow.github.io/luminaire/tutorial/streaming.html) for detailed information.
 
+## Examples
```

Comment:
What if we move this in the Quick Start section we have?

Reply:
I had it there initially but later decided to move since to have the context of batch and streaming before we have the examples.

Reply:
What if you add a hyperlink on the Quick Start to this Examples section?

Reply:
That's a great point. I can add that